src/components/ImageCard/index.js

```diff
@@ -58,7 +59,7 @@ class ImageCard extends PureComponent {
       <Grid container className={classnames('card', { 'card-open': end > now })} id={id}>
         <div>
           <Link to={`/details/${id}`} replace>
-            <img className="img card-image" src={link} alt='meme' />
+          <Meme link={link} memeClass='img card-image'></Meme>  
```

Comment:
Minor thing, can you change `<A></A>` to `<A />`?
Can you do this throughout this PR?